CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.0 (09/01/20)
```

Comment:
Might be a good idea to add a line or two about what steps are needed to migrate from 5 to 6

Reply:
I think the only step the change in defaults. It would be helpful though, along with the proposal for an error message 👍